src/Language/AST/Builder/Behavior/ParseAliasTrait.php

```diff
@@ -0,0 +1,24 @@
+<?php
```

Comment:
I removed this file because it's basically the same as the `ParseNameTrait`, so I added a `$fieldName` argument to that trait instead.